src/entities/message-template/message-template.ts

```diff
@@ -43,6 +43,45 @@ export interface MessageTemplateRawPayloadCarousel {
   items?: MessageTemplateRawPayloadCarouselItem[]
 }
 
+export interface MessageTemplateRawPayloadQuickReply {
+  type?: 'text' | string | null
```

Comment:
What does "text" do when it can be a string anyhow? :)

Reply:
Fixed, thanks for pointing out. Apparently `text` is the only legal value for the type, but `string` looks more future-proof.